command_function.py

```diff
@@ -0,0 +1,59 @@
+import os
+from abc import ABCMeta,  abstractstaticmethod
+import json
+from schema_keys import keys
+
+class Icommand(metaclass=ABCMeta):
+    @abstractstaticmethod
+    def excute():
+        """the interface Icommand"""
+
+class commandfactory:
+    @staticmethod
+    def build_command(args):
+        command_type=args.command
+        if command_type == "create":
+            return create(args.schema)
+        elif command_type == "delete":
+            return delete()
+        elif command_type == "get":
+            return get()
+        elif command_type == "set":
+            return set()
+        
+class create(Icommand):
```

Comment:
- class name should start with Capital letter 
- make every class in a file
- put al commands classes in one folder

Reply:
Ok, I will handle this.